production/ksonnet/README.md

```diff
@@ -54,19 +57,36 @@ promtail + {
 ```
 Notice that `container_root_path` is your own data root for docker daemon, use `docker info | grep "Root Dir"` to get it.
 
-Then do `ks show loki` to see the manifests that'll be deployed to your cluster.
-Apply them using `ks apply loki`.
+As a last step, fill add the correct `spec.apiServer` and `spec.namespace` to `environments/default/spec.json`:
```

Comment:
When using the created `loki` environment, `spec.namespace` is already set correctly.

To avoid pasting config objects, rather use the canonical command to set properties:
```bash
$ tk set environments/loki --server=https://127.0.0.1:6443 # or whatever ip you want
```

Reply:
It would be better.

Reply:
What would be better and why? 😕